bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -509,33 +511,6 @@ limitations under the License.
           </execution>
         </executions>
       </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-failsafe-plugin</artifactId>
-        <configuration>
-          <!-- disable integration tests by default, opt-in via profile -->
-          <skip>true</skip>
```

Comment:
can you confirm tests are running as expected?

Reply:
yep